deps/rabbitmq_ct_helpers/src/rabbit_ct_broker_helpers.erl

```diff
@@ -1008,6 +1008,31 @@ stop_rabbitmq_nodes(Config) ->
           fun(NodeConfig) ->
                   stop_rabbitmq_node(Config, NodeConfig)
           end),
+    %% Except if disabled, we search for crashes logged in the test nodes after
+    %% they are stopped. If we find some, we log them again in the common_test
+    %% logs and throw an exception to make the test fail.
+    FindCrashes = case rabbit_ct_helpers:get_config(Config, find_crashes) of
+                      true ->
+                          true;
+                      false ->
+                          false;
+                      undefined ->
+                          case os:getenv("FIND_CRASHES") of
+                              undefined -> true;
```

Comment:
`undefined` can never be returned causing this new crash detection facility to be disabled by default.

Reply:
Good catch. The overall patch was incorrectly ported from the Khepri branch anyway...

I opened #9373 to fix this problem and another one.